src/util/mapbox.js

```diff
@@ -267,7 +298,7 @@ export class MapLoadEvent extends TelemetryEvent {
         // mapbox tiles.
         if (config.ACCESS_TOKEN &&
             Array.isArray(tileUrls) &&
-            tileUrls.some(url => isMapboxHTTPURL(url))) {
+            tileUrls.some(url => isMapboxURL(url) || isMapboxHTTPURL(url))) {
```

Comment:
Is this `isMapboxURL` bit a separate change, or am I missing how this is connected to the rest of the PR?

Reply:
It is separate I guess. https://github.com/mapbox/mapbox-gl-js/pull/7594 regressed the ability to detect mapbox tile URLs.

Reply:
Ah, got it.